Common/src/main/resources/assets/jei/lang/en_us.json

```diff
@@ -126,38 +124,38 @@
+  "config.jei.search.searchAdvancedTooltips": "Search in Advanced Tooltips",
   "config.jei.advanced": "Advanced",
   "config.jei.advanced.comment": "Advanced config options to change the way JEI functions.",
   "config.jei.advanced.itemBlacklist": "Ingredient Blacklist",
-  "config.jei.advanced.itemBlacklist.comment": "List of ingredients that should not be displayed in the ingredient list overlay. Format: modId[:name[:meta]]. Hide Ingredients Mode will automatically add or remove entries here.",
+  "config.jei.advanced.itemBlacklist.comment": "List of ingredients that should not be displayed in the ingredient list overlay. Format: modId[:name[:meta]]. The hide ingredients mode will automatically add or remove entries here.",
   "config.jei.advanced.colorSearchEnabled": "Enable Color Search",
-  "config.jei.advanced.colorSearchEnabled.comment": "Search ingredients by color and show the colors on tooltips in the ingredient list overlay.",
+  "config.jei.advanced.colorSearchEnabled.comment": "Search ingredients by color and show the colors in tooltips in the ingredient list overlay.",
   "config.jei.advanced.debugModeEnabled": "Debug Mode",
   "config.jei.advanced.debugModeEnabled.comment": "Only useful for JEI developers, adds test ingredients and some debug recipes.",
   "config.jei.advanced.centerSearchBarEnabled": "Center Search Bar",
   "config.jei.advanced.centerSearchBarEnabled.comment": "Move the JEI search bar to the bottom center of the screen.",
   "config.jei.advanced.modNameFormat": "Mod Name Format",
-  "config.jei.advanced.modNameFormat.comment": "How the mod name should be formatted in the tooltip for JEI GUIs. Leave blank to disable.",
+  "config.jei.advanced.modNameFormat.comment": "Formatting for the mod names in tooltips for JEI GUIs. Leave blank to disable.",
   "config.jei.advanced.maxColumns": "Max Overlay Width",
   "config.jei.advanced.maxColumns.comment": "The maximum width of the ingredient and bookmark list overlays.",
   "config.jei.advanced.maxRecipeGuiHeight": "Max Recipe GUI Height",
   "config.jei.advanced.maxRecipeGuiHeight.comment": "The maximum height of the recipe GUI.",
   "config.jei.advanced.giveMode": "Give Mode",
-  "config.jei.advanced.giveMode.comment": "Choose if JEI should give ingredients direct to the inventory (inventory) or pick them up with the mouse (mouse_pickup).",
+  "config.jei.advanced.giveMode.comment": "Choose if JEI should give ingredients directly to the inventory (inventory) or pick them up with the mouse (mouse_pickup).",
   "config.jei.advanced.addBookmarksToFront": "Prepend New Bookmarks",
-  "config.jei.advanced.addBookmarksToFront.comment": "When true, adds new bookmarks to the front of the bookmark list. When false, adds new bookmarks to the end of the bookmark list",
+  "config.jei.advanced.addBookmarksToFront.comment": "Add new bookmarks to the front of the bookmark list instead of the end.",
```

Comment:
This entire section also qualifies for deletion, as these text strings don't seem to have been used since the release of JEI for Minecraft 1.13. However, since it's quite a large chunk, I preferred not to touch it before consulting with you first. What's your opinion on this?

Reply:
I forgot to implement support for these in the new config system.
I need to add support for localization in client configs, so let's leave these here.